### ISSUE TITLE
Uncaught RangeError: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by minzdrav** [phphe/virtual-list#2]
Hello @phphe 
Virtual list throw exception:
![image](https://user-images.githubusercontent.com/1764392/225867355-7f9478ba-2533-4315-9630-67d34c7eefea.png)

Please try to replace 1000 by something like 2000 or 3000 in App.vue.

```
export default defineComponent({
  components: { "v-list": vList },
  data() {
    return {
      items: new Array(3000).fill(1).map((v, i) => ({
        text: "Item " + i,
        lineHeight: 20 + (i % 20) + "px",
        width: 100 + (i % 30) + "px",
      })),
    };
  },
  mounted() {},
});
</script>
```

**Comment by minzdrav** [comment]
![image](https://user-images.githubusercontent.com/1764392/225869022-13991efe-a8f4-4693-889c-660fcd7ba9c1.png)
It's easy to fix. You shouldn't use positions inside positions.

**Comment by phphe** [comment]
ok, I will try to fix

**Comment by phphe** [comment]
fixed in 1.2.0

**Comment by minzdrav** [comment]
Thank you!
